### PR TITLE
Added LmgtfyCommand.java

### PR DESCRIPTION
**Pull request by Joshix-1**
This command helps users search the internet. It'll help resolving issues, which aren't complicated and just need a quick search.

### CODE REVIEW

**Comment by Vampire** [comment]
I'm sorry, but there are mutliple problems I have with this PR.
- I looked at lmgtfy and I really dislike it by now. It used to be a very nice service, but now it is bloated with ads and not nice and to the point like it used to be
- The "g" in "lmgtfy" stands for "google", but the link shows how to use DuckDuckGo. So either the command needs an argument so that everyone can generate the link to the engine he prefers with the default being Google as the name implies or different aliases like "lmddgtfy" or whatever
- I'm generally not too much of a fan of lmgtfy commands as they promote sending off people to a search enigne instead of trying to help. There are sure cases that should be simply googled and you can always make a lmgtfy link by hand and post it, but such a command is just too tempting often imho. :-D

@Saladoc what do you think?

**Comment by Saladoc** [comment]
Honestly, I'm against such a command. If we include one, it should be more neutrally named and do more than just generate a link.

In my opinion / experience there's only three use cases where such commands are meaningfully used.

**1. Answering lazy people**
When answering a truly lazy person, even the step of creating the search link for them is too much. Those people need to learn that people on discord are not the helpful tool where you drop in a poorly phrased question and that produces links of varying usefulness. That helpful tool is a search engine, and they should be directed to directly use one, and not handed any links, including those to actual search results on a silver platter.

**2. Helping people who don't know how to phrase their problem**
These are the people who *would* use a search engine themselves but lack the terminology or understanding to properly use a search engine to find their solution. Sometimes they are unfamiliar with the language, sometimes unaware that the problem they're looking to solve already has a name. Sometimes it's an XY-Problem plaguing them. 
Those people should be helped not by telling them to search, but what to search. For that, a website or command with condescending undertones is the wrong way. Either there is a more neutrally named command (which also provides more functionality than a basic link generator) or they are just told "try searching for *more exact term than the one they were using*.

**3. People who actually don't know what a search engine is**
It's safe to assume that someone who has managed to find Javacord is somehow familiar with the concept of a search engine. For the other one in a billion cases, a proper explanation can be procured somewhere else. 

TLDR: 
On top of the inconsistencies and issues @Vampire already pointed out I so wall-of-text-fully expanded on, I consider this a rather basic novelty command that is out of scope for a Bot tailored to Javacord support. *If* we were to include a web search command I'd at least expect it to display stubs of some results to have any added value. 
I do not think this is a worthwhile addition and am unwilling to approve this pull request both in it's current and any similar forms.

**Comment by Vampire** [comment]
Thanks for your effort Joshix, but as Saladoc also disagrees, I'm closing this unmerged, sorry.
